modules/ROOT/partials/configuration/comments-tinycomments_access.adoc

```diff
@@ -0,0 +1,62 @@
+[[tinycomments-access]]
+== `tinycomments_access`
+
+*Type:* `+String+`
+
+*Possible values:* `'comment'`
+
+This option controls the editor's read-only state and comment functionality.
+
+[IMPORTANT]
+====
+* When `tinycomments_access` is set to `comment`, the editor will automatically switch to xref:editor-important-options.adoc#readonly[Readonly Mode].
+* Default behavior, when the `tinycomments_access` option is either not configured or set to an empty string, allows full editing access to both the editor and the comments.
+====
+
+=== Example: Basic Configuration
+
+[source,javascript]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'tinycomments',
+  toolbar: 'addcomment showcomments',
+  readony: true, // set the editor to read-only mode
+});
+----
+
+=== `tinycomments_access` set to undefined or an empty string and `readonly` set to `true`
+
+When `tinycomments_access` is set to **undefined** or an **empty string**, the editor will remain in read-only mode, and the comments will be in read-only mode. This means that users cannot edit the content or add comments.
```

Comment:
```suggestion
=== `tinycomments_access` is not configured and `readonly` set to `true`

When `tinycomments_access` is not configured, the editor will remain in read-only mode, and the comments will be in read-only mode. This means that users cannot edit the content or add comments.
```
The option could be not configured, could be any values that's invalid and the `readonly: true` will still overwrite the editor state. Maybe just not configured or invalid?